improver_tests/utilities/temporal/test_temporal.py

```diff
@@ -138,7 +138,7 @@ def test_basic(self):
         result = iris_time_to_datetime(self.cube.coord("time"))
         self.assertIsInstance(result, list)
         for item in result:
-            self.assertIsInstance(item, datetime)
+            self.assertIsInstance(item, cftime.DatetimeGregorian)
```

Comment:
Is this the right fix or should we modify the iris_time_to_datetime code to actually return a datetime?
See what Max did here: https://github.com/metoppv/improver/pull/2137

Reply:
I've made a minor edit to return a python datetime object from the iris_time_to_datetime function.